recipe/meta.yaml

```diff
@@ -36,6 +37,7 @@ requirements:
     - snuggs >=1.4.1
     - click-plugins
     - enum34  # [py<34]
+    - proj4 <6
```

Comment:
Bummer that `rasterio` cannot be installed with the rest of the stack that is already using `proj4 6` :-(

Reply:
Yeah, that's too bad. What was the reason for the incompatibility?

Reply:
There are 7 tests failing when we use `proj4 6` (check the previous commit build logs). I'll raise it upstream.

Reply:
It looks like the reproject tests are failing. I wonder if it has to do with the updates to PROJ and it's integration with GDAL. Definitely something to dig into. Thanks for raising the issue upstream!